lib/super_good/solidus_taxjar/api_params.rb

```diff
@@ -116,7 +116,7 @@ def line_items_params(line_items)
             line_items: valid_line_items(line_items).map do |line_item|
               {
                 id: line_item.id,
-                quantity: line_item.quantity,
+                quantity: line_item.inventory_units.where.not(state: ['returned', 'canceled']).count,
```

Comment:
Since we're doing this a few times, should we consider pulling it into a helper so we don't have to repeat it?